spec/support/firebase_helper.rb

```diff
@@ -0,0 +1,46 @@
+module FirebaseHelper
+  def firebase_users
+    firebase_client.get('users_authenticated')
+  end
+
+  def firebase_find_users(email)
+    if firebase_users && firebase_users.body.present?
+      firebase_users.body.values.select {|u| u['email'] == email}
```

Comment:
we can use `firebase_user_emails` function here and then filter email. Additional check can be prevented

Reply:
@prashuchaudhary I want the whole user JSON here, because I need to compare various other fields in the User hash in the specs. Using `firebase_user_emails` will give me just the email.